config/routes.rb

```diff
@@ -3,7 +3,7 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
   resources :systems, :auth, :translations, :drafts, :resources, :custom_pages, :languages, :attributes,
-            :translated_attributes, :pages, :views, :attachments, :translated_pages, :follow_ups
+            :translated_attributes, :pages, :views, :attachments, :translated_pages, :follow_ups, :resource_types
```

Comment:
Should this one be `, only: [:index]` ?

If so, are there any other routes you should be restricting?